LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentClient.java

```diff
@@ -126,6 +126,7 @@ public void onResult(@Nullable Configuration configuration, @Nullable Exception
                             @Override
                             public void onResult(@Nullable LocalPaymentResult localPaymentResult, @Nullable Exception error) {
                                 if (localPaymentResult != null) {
+                                    braintreeClient.setPayPalContextID(localPaymentResult.getPaymentId());
```

Comment:
On iOS we took the approach of passing this into `sendAnalyticsEvent` as an optional null defaulted parameter ([see iOS PR 1181](https://github.com/braintree/braintree_ios/pull/1181). Not sure if we want to make this consistent across platforms.

Reply:
Yes, I am aware of that, I was reviewing how it works on iOS, the approach is very different on Android, no parameters are passed for the sendEvents method. I understand that we should be consistent between platforms, however I think that at this point it would be a bit strange, I think we have the flexibility to adapt the platforms. However, I am open to any instructions. 

Reference (in case you like to take a look at how it is implemented in Android): https://github.com/braintree/braintree_android/blob/ab5f70499890c147654ed4b94122447d29775a70/BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt#L213